packages/http-client-csharp/emitter/src/index.ts

```diff
@@ -2,7 +2,8 @@
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
 export { configurationFileName, tspOutputFileName } from "./constants.js";
-export { $onEmit } from "./emitter.js";
+export { $onEmit, CSharpEmitterContext, transformJSONProperties } from "./emitter.js";
```

Comment:
exporting `transformJSONProperties` looks quite temporary - could we do not export it and just copy its implementation over to autorest.csharp?

Reply:
I'd rather not duplicate the code if we can avoid it. It is exported as a beta function so it can break.

Reply:
An alternative would be to export a writeCodeModel function.